63.md

```diff
@@ -0,0 +1,90 @@
+NIP-63
+======
+
+Interactive Stories
+-------------------
+
+`draft` `optional`
+
+This NIP introduces `kind:30296` and `kind:30297` as living notes designed to create interactive stories. These notes include `option` tags for navigating to subsequent notes, allowing readers to continue exploring the story. `Kind:30296` serves as starting events for the reader, while `kind:30297` represents individual scenes within the narrative.
+
+The modularity of this approach allows Scenes from multiple authors to be combined into bigger stories. 
+
+### Prologue (kind:30296)
+
+Prologue events are entry points in the story. 
+
+They SHOULD contain `title`, `summary` and `image` to help clients render a preview of the story.
+
+`option` tags include a description for the action button, an address and the relay to find the next scene.
+
+```jsonc
+{
+  "kind": 30296,
+  "content": "<the beginning of the story>",
+  "tags": [
+    ["d", "<unique identifier>"],
+    ["title", "<short title for the prologue>"],
+    ["summary", "<summary for preview>"],
+    ["image", "<image url for preview>"],
+
+    ["option", "<description>", "<kind:pubkey:d-tag>", "<relay_hint>"],
+    ["option", "<description>", "<kind:pubkey:d-tag>", "<relay_hint>"],
+    // ...
+  ]
+}
+```
+
+### Scene (kind:30297)
+
+Scenes describe a given point in a story with potential options and SHOULD include an optional `title` tag.
+
+```jsonc
+{
+  "kind": 30297,
+  "content": "<description of the scene>",
+  "tags": [
+    ["d", "<unique identifier>"],
+    ["title", "<short title for this scene>"],
+
```

Comment:
Make it possible to reply to scenes with further scenes. If your story gets referenced, your client should pick up on that (if you follow me for example) and make it easy for you to add my event as an option using the title as description of a further option.

```suggestion
    ["title", "<short title for this scene>"],
    ["E", "<root_event_id>", "<relay_hint>"],
    ["e", "<event_id>", "<relay_hint>"],
```

Reply:
hum... I wonder how to do that.. we could add a tag but then you will have to remember deleting it later. Or maybe we just create another kind...